openshift_scalability/content/centos-stress/root/bin/run

```diff
@@ -189,19 +191,22 @@ main() {
       results_filename=jmeter-"${HOSTNAME}"-"$(date +%y%m%d%H%M)" 
 
       # Call JMeter packed with ENV vars
-      jmeter -n -t test.jmx -Jnum=${NUM} -Jramp=${JMETER_RAMP} \
+      jmeter.sh -n -t test.jmx -Jnum=${NUM} -Jramp=${JMETER_RAMP} \
         -Jduration=${RUN_TIME} -Jtpm=${JMETER_TPS} \
         ${ips} \
         -Jport=${TARGET_PORT} \
         -Jresults_file="${results_filename}".jtl -l "${results_filename}".jtl \
         -j "${results_filename}".log -Jgun="${GUN}" || die $? "${RUN} failed: $?"
 
-      have_server "${GUN}" && scp -o StrictHostKeyChecking=false -p *.jtl *.log *.png ${GUN}:${PBENCH_DIR}
+      if have_server "${SERVER_RESULTS}" ; then
```

Comment:
Sometimes you have a space before `;` but mostly you do not. Can we please keep it the same one way or another?

Reply:
Always use a space in the context you're referring to.  The only occurrence with no space is a script I adapted from another source.  Will change.